package.json

```diff
@@ -68,7 +68,8 @@
     "serve-favicon": "~2.3.2",
     "tail": "^1.2.1",
     "uuid": "^3.0.1",
-    "wordwrap": "1.0.0"
+    "wordwrap": "1.0.0",
+    "xhr2": "^0.1.3"
```

Comment:
`xhr2` is a new dependency for http-pouchdb: https://github.com/pouchdb/http-pouchdb/blob/9b28231f2c40795c045956a6a4b0f3f974c661c9/package.json#L51